ts/screens/portfolio/AddCardScreen.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+    Button,
+    Col,
+    Content,
+    Grid,
+    Icon,
+    Input,
+    Item,
+    Text
+} from "native-base";
+import * as React from "react";
+import {
+  FlatList,
+  Image,
+  View
+} from "react-native";
+import { NavigationScreenProp, NavigationState} from "react-navigation";
+
+import I18n from "../../i18n"
+import ROUTES from "../../navigation/routes"
+
+type Props = {
+    navigation: NavigationScreenProp<NavigationState>
+}
+
+type State = {
+    cardNumber: string,
+    expireDate: string,
+    secureCode: string,
+    cardHolder: string
+}
+
+export class AddCardScreen extends React.Component<Props, State> {
+
+  public static navigationOptions = {
+      title: I18n.t("portfolio.addCardTitle"),
+      headerBackTitle: null
+  }
+
+  constructor(props: Props) {
+    super(props)
+    this.state = {
+        cardNumber: "",
+        expireDate: "",
+        secureCode: "",
+        cardHolder: ""
+    }
+  }
+
+  public render(): React.ReactNode {
+
+    const cardItems = [
+        {name:"amexIcon",       source: require("../../../img/portfolio/cards-icons/amex.png")},
+        {name:"dinersIcon",     source: require("../../../img/portfolio/cards-icons/diners.png")},
+        {name:"maestroIcon",    source: require("../../../img/portfolio/cards-icons/maestro.png")},
+        {name:"mastercardIcon", source: require("../../../img/portfolio/cards-icons/mastercard.png")},
+        {name:"postepayIcon",   source: require("../../../img/portfolio/cards-icons/postepay.png")},
+        {name:"visaIcon",       source: require("../../../img/portfolio/cards-icons/visa.png")},
+        {name:"visaEIcon",      source: require("../../../img/portfolio/cards-icons/visa-electron.png")},
+    ];
+
+    return(
+    <Content>
+        <Item style={{borderBottomWidth: 0, marginTop: 5, marginRight: 5, marginLeft: 5,padding:0}}>
+            <Text>{I18n.t("portfolio.dummyCard.labels.name")}</Text>
+        </Item>
+        <Item style={{borderBottomWidth: 3, borderBottomColor:"#000000", marginRight: 5, marginLeft: 5}}>
```

Comment:
I suggest to use only a predefined set of margin / border values that comes from theme paramters.
cc @fpersico

Reply:
cc @valentinacala thanks :)